roles/usegalaxy-eu.galaxy-procstat/templates/telegraf-procstat.conf.j2

```diff
@@ -2,7 +2,7 @@
 
 {% for guniconrn in range(galaxy_systemd_gunicorns) %}
 [[inputs.procstat]]
-	cgroup = "systemd/system.slice/system-galaxy\\x2dgunicorn.slice/galaxy-gunicorn@{{ zerg }}.service"
+	cgroup = "systemd/system.slice/system-galaxy\\x2dgunicorn.slice/galaxy-gunicorn@{{ guinicorn }}.service"
```

Comment:
`guinicorn` (extra i)? vs loop variable `guniconrn` (extra n)

Reply:
Yes I just saw it in the Jenkins fail message, sorry :/